classes/system/keepalived/cluster/instance/stacklight_monitor_vip.yml

```diff
@@ -0,0 +1,16 @@
+applications:
```

Comment:
this is basically service level, for this just include service keepalived and params for interface and router id [1...255] as with nova.cluster.yml for example